### DISCUSSION TITLE
Web Research Retriever

### DISCUSSION THREAD

**Discussion by natellanur** [FlowiseAI/Flowise#1183] [Ideas]
Hey guys,
I was wondering whether WebResearchRetriever is in the roadmap? Would love to see this feature to create chains of web search (so that the agent was able to follow multiple links from google search to give better results).
Or any ideas how I could do that now?
Thanks! :)

**Comment by HenryHengZJ** [comment]
Dont think JS has this yet only in Python, for reference - https://python.langchain.com/docs/modules/data_connection/retrievers/web_research
